hugo/content/en/examples/shortcodes/table/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Table
+weight: 29
+---
+
+## Basic table
+
+```
+{{</* table */>}}
+| Syntax      | Description |
+| ----------- | ----------- |
+| Header      | Title       |
+| Paragraph   | Text        |
+{{</* /table */>}}
+```
+
+```
+{{</* table */>}}
+| Syntax | Description |
+| --- | ----------- |
+| Header | Title |
+| Paragraph | Text |
+{{</* /table */>}}
+```
+
+Both versions result in the following table
```

Comment:
Is the difference between both examples just formatting/readability?

Reply:
Yes